sound/soc/intel/boards/cml_rt5682_max98357a.c

```diff
@@ -0,0 +1,438 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
I don't want to follow the same route as before. We want to reuse machine drivers, so please remove references to cml.